apps/mannschaft/pages/_app.tsx

```diff
@@ -152,8 +158,27 @@ function CustomApp({Component, pageProps, emotionCache}: CustomAppProps) {
                 <NavBar
                   categorySlugs={[['categories', 'other'], ['about-us']]}
                   slug="main"
-                  headerSlug="header"
-                />
+                  headerSlug="header">
+                  <ButtonLink href="/search">
```

Comment:
This causes a full page reload technically, you can use Link from @wepublish/utils/website instead.